src/test-2.js

```diff
@@ -0,0 +1,52 @@
+const arrObjTask = [
```

Comment:
Please add `npm install`, `npm start`, and `npm test` commands in README for a non-technical user.
Please add all unnecessary files and folders like `vscode` to `gitignore`

Reply:
Thanks for the sugestion. Implementing now.